src/backup.rs

```diff
@@ -217,27 +224,40 @@ impl BackupEngine {
 }
 
 impl BackupEngineOptions {
-    //
-}
+    pub fn new<P: AsRef<Path>>(backup_dir: P) -> Result<Self, Error> {
```

Comment:
Please, add some description to the public methods.

Reply:
Done. Please have another look.